src/sentry/static/sentry/app/components/sidebar/index.tsx

```diff
@@ -624,29 +645,31 @@ class Sidebar extends React.Component {
   }
 }
 
-const SidebarContainer = createReactClass({
-  displayName: 'SidebarContainer',
-  mixins: [Reflux.listenTo(PreferencesStore, 'onPreferenceChange')],
-  getInitialState() {
-    return {
-      collapsed: PreferencesStore.getInitialState().collapsed,
-    };
-  },
-
-  onPreferenceChange(store) {
-    if (store.collapsed === this.state.collapsed) {
-      return;
-    }
+const SidebarContainer = ReactRouter.withRouter(
```

Comment:
Hmm, my conversion here is from a somewhat old stash, so maybe we removed react router recently.

Reply:
Ahh I see billy did this here 84df9403ff33106cc8a43d30966efa5209eb7a20